changelog/Microsoft.DirectoryServices.json

```
@@ -1,6 +1,32 @@
 {
   "changelog": [	
     {
+      "ChangeList": [
+        {
+          "Id": "3dab5ede-46c2-4ee7-b8ce-bc09b7299e3f",
+          "ApiChange": "Property",
+          "ChangedApiName": "links",
+          "ChangeType": "Addition",
+          "Description": "Added the **links** property to [synchronizationJobSubject](https://docs.microsoft.com/en-us/graph/api/resources/synchronizationJobSubject?view=graph-rest-beta) resource.",
+          "Target": "synchronizationJobSubject"
+        },
+        {
+          "Id": "3dab5ede-46c2-4ee7-b8ce-bc09b7299e3f",
+          "ApiChange": "Resource",
+          "ChangedApiName": "synchronizationLinkedObjects",
+          "ChangeType": "Addition",
+          "Description": "Added the [synchronizationLinkedObjects](https://docs.microsoft.com/en-us/graph/api/resources/synchronizationLinkedObjects?view=graph-rest-beta) resource type.",
+          "Target": "synchronizationLinkedObjects"
+        }
+      ],
+      "Id": "3dab5ede-46c2-4ee7-b8ce-bc09b7299e3f",
+      "Cloud": "Prod",
+      "Version": "beta",
+      "CreatedDateTime": "2021-12-03T06:27:32.7966576Z",
+      "WorkloadArea": "Identity and access",
+      "SubArea": "Directory management"
```

Minor fix - the Synchronization APIs lie in the "Applications" node
```suggestion
      "WorkloadArea": "Applications",
      "SubArea": ""
```